tests/test_vision_classifier.py

```diff
@@ -33,6 +34,12 @@
     columns=["path", "label", "fold"],
 )
 
+NUMPY_IMAGE1 = Image.open(f"{TEST_PATH}/resources/images/val/cat/cat3.jpg")
+NUMPY_IMAGE1 = np.array(NUMPY_IMAGE1.resize((800, 800)), "uint8")
+NUMPY_IMAGE2 = Image.open(f"{TEST_PATH}/resources/images/val/dog/dog3.jpg")
+NUMPY_IMAGE2 = np.array(NUMPY_IMAGE2.resize((800, 800)), "uint8")
```

Comment:
```suggestion
def numpy_image_from_jpg(path: str) -> np.ndarray:
    image = Image.open(path)
    numpy_image = np.array(image.resize((800, 800)), "uint8")
    return numpy_image
    
NUMPY_IMAGE_1 =numpy_image_from_jpg(f"{TEST_PATH}/resources/images/val/cat/cat3.jpg")
NUMPY_IMAGE_2 =numpy_image_from_jpg(f"{TEST_PATH}/resources/images/val/dog/dog3.jpg")
```

Reply:
Done, I have created the function in utils